tests/unit/FirebaseArray.spec.js

```diff
@@ -28,11 +28,11 @@ describe('$FirebaseArray', function () {
     }
   };
 
-  var $firebase, $fb, $fbOldTodo, arr, $FirebaseArray, $utils, $rootScope, $timeout, destroySpy, testutils;
+  var $fbOldTodo, arr, $FirebaseArray, $utils, $rootScope, $timeout, destroySpy, testutils;
```

Comment:
`$fbOldTodo`, `$rootScope`, and `destroySpy` are not used anywhere in this file, so we can remove them as well.